src/app/component/MyAccountCreateAccount/MyAccountCreateAccount.component.js

```diff
@@ -16,15 +16,41 @@ import Field from 'Component/Field';
 import Form from 'Component/Form';
 import { signInStateType } from 'Type/Account';
 
+import { SHOW_VAT_NUMBER_REQUIRED } from './MyAccountCreateAccount.config';
+
 /** @namespace Component/MyAccountCreateAccount/Component */
 export class MyAccountCreateAccount extends PureComponent {
     static propTypes = {
         state: signInStateType.isRequired,
         onCreateAccountAttempt: PropTypes.func.isRequired,
         onCreateAccountSuccess: PropTypes.func.isRequired,
-        handleSignIn: PropTypes.func.isRequired
+        handleSignIn: PropTypes.func.isRequired,
+        showTaxVatNumber: PropTypes.string.isRequired
     };
 
+    renderVatNumberField() {
+        const { showTaxVatNumber } = this.props;
+        const validation = [];
+
+        if (!showTaxVatNumber) {
+            return null;
+        }
+
+        if (showTaxVatNumber === SHOW_VAT_NUMBER_REQUIRED) {
+            validation.push('notEmpty');
+        }
```

Comment:
I would rather go for vatNumberValidation coming from container.

Reply:
Done.